src/radical/pilot/agent/rm/loadleveler.py

```diff
@@ -58,7 +56,7 @@ class LoadLeveler(LRMS):
         29: {'A':  0, 'B': 30, 'C': 28, 'D': 17, 'E': 26},
         30: {'A':  3, 'B': 29, 'C': 31, 'D': 18, 'E': 25},
         31: {'A':  2, 'B': 28, 'C': 30, 'D': 19, 'E': 24},
-        }
+                     }
 
```

Comment:
Uh?

Reply:
It was the wrong indentation. Based on flake8 and pylint closing should be at the same column as opening.

Reply:
But this is much worse than it was before.  If this needs fixing, then the *data* need to be shifted, too.

Reply:
Yes, they should!